### PR TITLE
Add board lilygo_ttgo_t-01c3

### PR DESCRIPTION
**Pull request by prplz**
https://github.com/Xinyuan-LilyGO/T-01C3

I'm not sure where to get the bluetooth creation/creator IDs

### CODE REVIEW

**Comment by prplz** [comment]
Force pushed to fix debug uart change (83593a1558c2e00be13a62e502fcb9ee923a9d6f)

**Comment by tannewt** [comment]
Creator ids are documented here: https://github.com/creationid/creators Please make one for LilyGo on their behalf and we could make a second repo to list the creation ids for them. We don't actually need them involved. It's just good to document them somewhere.

**Comment by prplz** [comment]
Done: https://github.com/creationid/creators/pull/6

**Comment by tannewt** [comment]
Ready for review?

File: ports/espressif/boards/lilygo_ttgo_t-01c3/mpconfigboard.mk [hunk 1]
```diff
@@ -0,0 +1,10 @@
+CIRCUITPY_CREATOR_ID = 0xC3C3_0000
+CIRCUITPY_CREATION_ID = 0x00C3_0001
```
Comment (tannewt) [inline ports/espressif/boards/lilygo_ttgo_t-01c3/mpconfigboard.mk:2]:
```suggestion
CIRCUITPY_CREATION_ID = 0x00C30001
```

**Review by tannewt — APPROVED** [review]
Thank you!

**Pull request merged**